street_network/src/transform/classify_intersections.rs

```diff
@@ -34,8 +33,50 @@ fn guess_complexity(_inter: &Intersection, roads: Vec<OriginalRoad>) -> Intersec
         return Connection;
     }
 
-    // A MultiConnection is characterised by exactly two connected dividing lines (lines that
-    // separate traffic in different directions) and no traffic that crosses them.
+    // A MultiConnection is characterised by exactly one dividing line traveling through it (the
+    // line that separates traffic in different directions) and no traffic that crosses it.
+    if roads.len() == 3 {
```

Comment:
Check out https://www.openstreetmap.org/node/608494028 in the `arizona_highways` test. This feels like a case too -- one main uninterrupted road, two ramps merging in on either side

Reply:
![pxl_20220920_035821523_720](https://user-images.githubusercontent.com/1664407/191164438-6dca4c8b-f7b1-4de9-899c-743240c00257.jpg)
Hmm, the three movements conflict in the sense that they all have the same destination. So maybe we do need geometry to distinguish this from the T intersection case?

... I'm pretty sure you mentioned an idea about looking at roads in clockwise order and seeing what adjacent pairs look like. Does that help us at all maybe? Or do we need to generate the polyline of the movement and look for intersections?

Reply:
https://github.com/a-b-street/osm2streets/issues/67#issuecomment-1251012734
Ah, you're way ahead of me